tools/refguide_check.py

```diff
@@ -331,6 +420,19 @@ def check_items(all_dict, names, deprecated, others, module_name, dots=True):
 
 
 def validate_rst_syntax(text, name, dots=True):
+    """
+    Validates the doc string text of a module `name`
+    Parameters
+    ----------
+    text : str
+        Docstring of module
+    name : str
+        name of module
+
```

Comment:
dots: Boolean, whether to print a symbol for each check

Reply:
Rectified